### PR TITLE
eng-1828 add spacing above the failed-to-load error

### PR DESCRIPTION
**Pull request by eunice-chan**
## Describe your changes and why you are making these changes
Fix spacing.
- Add spacing to the top of the sidesheet
- Fix width so error does not go off screen.

## Related issue number (if any)
eng-1828

## Loom demo (if any)
**Before: Little/no space at top of sidesheet**
Screen Shot 2022-10-13 at 8 35 54 AM
Screen Shot 2022-10-13 at 8 35 41 AM

**After: More space at top of sidesheet**
Screen Shot 2022-10-13 at 8 31 45 AM
Screen Shot 2022-10-13 at 8 32 22 AM

**Before: Error goes off screen**
Screen Shot 2022-10-13 at 8 36 19 AM

**After: Error ends on-screen**
Screen Shot 2022-10-13 at 8 31 54 AM

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [N/A] If this is a new feature, I have added unit tests and integration tests.
- [N/A] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/components/pages/artifact/id/index.tsx [hunk 1]
```diff
@@ -85,11 +85,11 @@ const ArtifactDetailsPage: React.FC<ArtifactDetailsPageProps> = ({
 
   const { metrics, checks } =
     !!workflowDagResultWithLoadingStatus &&
-      isSucceeded(workflowDagResultWithLoadingStatus.status)
```
Comment (eunice-chan) [inline src/ui/common/src/components/pages/artifact/id/index.tsx:88]: from lint

File: src/ui/common/src/components/pages/artifact/id/index.tsx [hunk 2]
```diff
@@ -122,8 +122,9 @@ const ArtifactDetailsPage: React.FC<ArtifactDetailsPageProps> = ({
   useEffect(() => {
     if (!!artifact) {
       if (!sideSheetMode) {
-        document.title = `${artifact ? artifact.name : 'Artifact Details'
```
Comment (eunice-chan) [inline src/ui/common/src/components/pages/artifact/id/index.tsx:125]: also from lint

File: src/ui/common/src/components/pages/metric/id/index.tsx [hunk 3]
```diff
@@ -121,8 +121,9 @@ const MetricDetailsPage: React.FC<MetricDetailsPageProps> = ({
   useEffect(() => {
     if (!!operator && !sideSheetMode) {
       // this should only be set when the user is viewing this as a full page, not side sheet.
-      document.title = `${operator ? operator.name : 'Operator Details'
```
Comment (eunice-chan) [inline src/ui/common/src/components/pages/metric/id/index.tsx:124]: lint

File: src/ui/common/src/components/pages/metric/id/index.tsx [hunk 4]
```diff
@@ -154,7 +155,7 @@ const MetricDetailsPage: React.FC<MetricDetailsPageProps> = ({
       .map(
         (artifactId) =>
           (workflowDagResultWithLoadingStatus.result?.artifacts ?? {})[
-          artifactId
```
Comment (eunice-chan) [inline src/ui/common/src/components/pages/metric/id/index.tsx:157]: linter

File: src/ui/common/src/components/pages/operator/id/index.tsx [hunk 5]
```diff
@@ -197,7 +197,7 @@ const OperatorDetailsPage: React.FC<OperatorDetailsPageProps> = ({
       .map(
         (artifactId) =>
           (workflowDagResultWithLoadingStatus.result?.artifacts ?? {})[
-          artifactId
```
Comment (eunice-chan) [inline src/ui/common/src/components/pages/operator/id/index.tsx:200]: lint

**Review by vsreekanti — APPROVED** [review]

**Pull request merged**